shell/browser/electron_browser_main_parts.cc

```diff
@@ -386,7 +387,8 @@ void ElectronBrowserMainParts::PostDestroyThreads() {
 #endif
 #if BUILDFLAG(IS_LINUX)
   device::BluetoothAdapterFactory::Shutdown();
-  bluez::DBusBluezManagerWrapperLinux::Shutdown();
+  bluez::BluezDBusManager::Shutdown();
```

Comment:
This function `CHECK()`s that `bluez::BluezDBusManager::Initialize()` has been called first, so  I _think_ we need to wrap this with `if (bluez::BluezDBusManager::IsInitialized())`?

```suggestion
  if (bluez::BluezDBusManager::IsInitialized())
    bluez::BluezDBusManager::Shutdown();
```